spec/parliament/ntriple/utils_spec.rb

```diff
@@ -184,4 +184,24 @@
       expect(sorted_incumbencies[1].end_date).to eq(DateTime.new(2015, 3, 30))
     end
   end
+
+  describe '#multi_direction_sort' do
+    it 'returns a response sorted by member_count (desc) and name (asc)' do
```

Comment:
Just to check - is there an upper limit to the number of sorts and orders you will allow?

If so we should raise an error if more than that - if not we should document and maybe have like a 10 deep test for sanity checking